README.rst

```diff
@@ -1,28 +1,68 @@
 |Build Status| |Coveralls| |npm_version| |npm_downloads| |license|
 
 frontend-app-payment
-=========================
+====================
 
 Please tag **@edx/arch-team** on any PRs or issues.  Thanks.
 
 Introduction
 ------------
 
-React app for payment.
+Micro-frontend for the single-page payment/checkout process.
 
-Important Note
---------------
+Getting Started
+---------------
 
-The production Webpack configuration for this repo uses `Purgecss <https://www.purgecss.com/>`_ 
+1. Clone this repo:
+
+  ``git clone https://github.com/edx/frontend-app-payment.git``
+
+2. Install npm dependencies:
+
+  ``cd frontend-app-payment && npm install``
+
+3. Start the dev server:
+
+  ``npm start``
+
+The dev server is running at `http://localhost:1998 <http://localhost:1998>`_.
+
+By default it will show an empty basket view.
+
+Cart States
+-----------
+
+**Empty Cart**
+
+Visit `http://localhost:1998 <http://localhost:1998>`_ without adding any product to your cart.
+
+**Single Course Purchase**
+
+Assuming you have a fairly standard devstack setup, if you click the "Upgrade to Verified" button on the Demonstration Course on `http://localhost:18000/dashboard <http://localhost:18000/dashboard>`_, that will populate your cart with a single course so that you see the cart, order summary, and checkout form.
+
+**Other Types**
+
+*To be added.*
+
+Build Process Notes
+-------------------
+
+**Production Build**
+
+The production build is created with ``npm run build``.
+
+**Purgecss**
+
+The production Webpack configuration for this repo uses `Purgecss <https://www.purgecss.com/>`_
 to remove unused CSS from the production css file. In webpack/webpack.prod.config.js the Purgecss
 plugin is configured to scan directories to determine what css selectors should remain. Currently
 the src/ directory is scanned along with all @edx/frontend-component* node modules and paragon.
 If you add and use a component in this repo that relies on HTML classes or ids for styling you
-must add it to the Purgecss configuration or it will be unstyled in the production build. 
+must add it to the Purgecss configuration or it will be unstyled in the production build.
 
 
-Local Dev Server + Stage API
-----------------------------
+Appendix A: Local Dev Server + Stage API
+----------------------------------------
 
 If you would like to run this frontend against stage.edx.org you can run npm run start:stage and
 access your development server at https://local.stage.edx.org:1998 after the initial setup
```

Comment:
Are there plans afoot to make stage accessible only from the VPN?

Reply:
Not sure.